internal/proxy/makeproxy.sh

```diff
@@ -16,40 +16,19 @@
 # This script downloads and arranges packages in a structure for a
 # Go Modules proxy.
 
-set -o pipefail
+# https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
+set -euxo pipefail
 
 if [[ $# -ne 1 ]]; then
-  echo "usage: makeproxy.sh OUTDIR" 1>&2
+  echo "usage: makeproxy.sh <dir>" 1>&2
   exit 64
 fi
 
-export GO111MODULE=on
-GO="${GO:-go}"  # allows user customization of which Go command to use.
+GOPATH="$1"
```

Comment:
Also set `GOPROXY`?

Reply:
Leaving that out was intentional. The previous code set `GOPROXY` to the original file cache, which probably makes things faster, but also more fragile (e.g., if we now need to download some module that we haven't downloaded before). I don't care how fast this is so I'd rather do the simpler thing and let it download them.